src/webgpu/util/texture/texel_data.ts

```diff
@@ -28,13 +28,26 @@ type TexelComponentInfo = PerTexelComponent<{
   bitLength: number;
 }>;
 
-// A function which maps component values to component values.
+/**
+ * Maps component values to component values
+ * @param {PerTexelComponent<number>} components - The input components.
+ * @returns {PerTexelComponent<number>} The new output components.
```

Comment:
FYI it would have also been okay to omit params/returns if they're somewhat trivial (like here); `/** */` is still more useful than `//`. More documentation is better though.